src/Sentry/Protocol/Exceptions/SentryStackFrame.cs

```diff
@@ -248,6 +257,13 @@ public void WriteTo(Utf8JsonWriter writer)
                 writer.WriteNumber("symbol_addr", symbolAddress);
             }
 
+            // Instruction address
+            if (InstructionAddress is { } instructionAddress &&
+                Regex.IsMatch(instructionAddress, @"\A\b0[xX][0-9a-fA-F]+\b\Z"))
```

Comment:
if it doesn't match we just don't do anything? Should we match on the setter then?
Should this be a `long` instead?